index.coffee

```diff
@@ -17,7 +17,9 @@ module.exports = RedisSharelatex =
 			client.healthCheck = RedisSharelatex.singleInstanceHealthCheckBuilder(client)
 		else if opts.cluster?
 			Redis = require("ioredis")
-			client = new Redis.Cluster(opts.cluster)
+			standardOpts = _.clone(opts)
+			delete standardOpts.cluster
```

Comment:
We also often include a `key_schema` on this object which shouldn't get passed into the redis driver?